src/flick/tag/serializers.py

```diff
@@ -12,5 +12,5 @@ class TagSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Tag
-        fields = ("tag_id", "tag", "shows", "lsts")
+        fields = ("tag_id", "ext_api_id", "tag", "shows", "lsts")
```

Comment:
why was this added?

Reply:
I need a way to store the external api id of the genre so it would be a lot easier to convert our tag_id to check against the genre ids returned from external APIs